maven-surefire-common/src/main/java/org/apache/maven/plugin/surefire/booterclient/JarManifestForkConfiguration.java

```diff
@@ -150,8 +150,10 @@ private static String toClasspathElementUri( @Nonnull Path parent,
     {
         try
         {
-            return new URI( null, parent.relativize( classPathElement.toPath() ).toString(), null )
-                    .toASCIIString();
+            return new URI( null,
+                            parent.relativize( classPathElement.toPath() ).toString().
```

Comment:
You could use a separate variable since this is getting long line. It would be at least easy to debug.
```
Path absolutePath = classPathElement.toPath();
String relativeUriPath = parent.relativize( absolutePath )
.toString()
.replace( File.separatorChar, '/' );
return new URI( null, relativeUriPath, null ).toASCIIString();
```